spec/factories/applicants.rb

```diff
@@ -1,11 +1,11 @@
 FactoryBot.define do
   factory :applicant do
-    name { Faker::Name.full_name }
+    name { Faker::Name.name }
     address { Faker::Address.full_address }
     city { Faker::Address.city }
     state { Faker::Address.state_abbr }
     zip { Faker::Address.zip }
-    phonenumber { "MyString" }
+    phonenumber { Faker::PhoneNumber.cell_phone }
```

Comment:
Nice solution here. these Factory Bots are cool

Reply:
I figured it out. We are okay.